src/legacy/proza-message/src/lib/vl-proza-message.scss

```diff
@@ -13,67 +13,45 @@
     box-shadow: 0 0 0 1px inset rgba(255, 170, 51, 0.5);
 }
 
-::slotted(#wysiwyg) {
-    display: inline-block !important;
-    width: 100% !important;
-    line-height: 1em !important;
-}
-
-::slotted(#wysiwyg:focus) {
-    outline: 0 !important;
-    border-color: $vl-amber !important;
-    box-shadow: inset 0 0 0 0.1rem $vl-amber !important;
-    padding: 0.5rem 0 !important;
-    line-height: 1em !important;
+.vl-proza-message--updatable {
+    display: inline-flex;
+    align-items: center;
+    &>*:not(:last-child){
+        padding-right: 0.5em;
+    }
+    #actions {
+        display: inline-flex;
+        align-items: center;
+        &>*:not(:last-child){
+            margin-right: 0.3em;
+        }
+    }
 }
 
-::slotted(vl-typography) {
-    display: inline;
+:host(.vl-proza-message__block) .vl-proza-message--updatable {
+    align-items: end;
 }
 
-:host(.vl-proza-message__block) [is="vl-button-link"] {
-    position: absolute;
-    right: 4px;
-    bottom: 2px;
-}
+[is="vl-button"] {
+    cursor: pointer !important;
+    height: 1.5em !important;
+    width: 1.5em !important;
 
-[is="vl-button-link"] {
-    line-height: 1em !important;
-}
-
-[is="vl-icon"] {
-    font: icon !important;
-    vertical-align: middle !important;
-}
-
-[is="vl-button-link"] [is="vl-icon"] {
-    color: #ffaa33;
-    mix-blend-mode: difference;
+    background-color: $vl-amber !important;
```

Comment:
ik ben persoonlijk niet zo'n fan van die gele achtergrond maar dat is functioneel dus met Kenzo te bekijken.